demos/demo-minimal-js/README.md

```diff
@@ -0,0 +1,13 @@
+# Demo Widget - Minimal JS
+
+## Installation
+Use npm or yarn to install the app:
+```shell
+npm i
+```
+```shell
+npm run start
+```
+
+## Running the app
+View the app in https://localhost:9025/. Note that you may need bypass a "Your connection is not secure" warning in order to access the application.
```

Comment:
What do you think about linking to the "Launch the demo call widget from HubSpot" section in both demo READMEs?

Reply:
So the link would have to be https://github.com/HubSpot/calling-extensions-sdk/tree/project-demo-v1#run-the-demo-call-widget since this updates the project-demo-v1 branch. Do you think we should wait until the changes are in master?

Reply:
Waiting makes sense!